src_text/TCP_connections.txt

```diff
@@ -1,22 +1,22 @@
 
-In toxcore each client is connected to 3 relays even if there are no TCP peers and the onion is not needed. It may be better to only connect to those relays when toxcore is initializing as this is the only time when peers will connect to us via TCP relays we are connected to. Due to how the onion works, after the initialization phase where each peer is searched in the onion and then if they are found the info required to connect back (DHT pk, TCP relays) is sent to them, there should be no more peers connecting to us via TCP relays. This may be a way to further reduce load on TCP relays however more research is needed.
+In toxcore, each client is connected to 3 relays even if there are no TCP peers and the onion is not needed. It might be optimal to only connect to these relays when toxcore is initializing as this is the only time when peers will connect to us via TCP relays we are connected to. Due to how the onion works, after the initialization phase, where each peer is searched in the onion and then if they are found the info required to connect back (DHT pk, TCP relays) is sent to them, there should be no more peers connecting to us via TCP relays. This may be a way to further reduce load on TCP relays, however, more research is needed before it is implemented.
 
-TCP_connections picks one relay and uses it and only it to send data to the other peer. The reason for not picking a random connected relay for each packet is that it severely deteriorates the quality of the link between two peers and makes performance of lossy video and audio transmissions really bad. For this reason one peer is picked and used to send all data. If for any reason no more data can be sent to that relay, the next relay is used. Reasons for that include the TCP socket being full which means that the relay should not be dropped if this occurs. Relays are only dropped if they time out, if they become useless (if the relay is one too many or is no longer used to relay data to any peers).
+TCP_connections picks one relay and uses it, and only it, to send data to the other peer. The reason for not picking a random connected relay for each packet is that it severely deteriorates the quality of the link between two peers and makes performance of lossy video and audio transmissions really poor. For this reason, one peer is picked and used to send all data. If for any reason no more data can be sent through that relay, the next relay is used. This may happen if the TCP socket is full and so the relay should not necessarily be dropped if this occurs. Relays are only dropped if they time out or if they become useless (if the relay is one too many or is no longer being used to relay data to any peers).
```

Comment:
>TCP_connections picks one relay and uses it, and only it, to send data (…)

perhaps change to ``TCP_connections picks one relay and uses only it to send data (…)`` ?

>This may happen if the TCP socket is full and so the relay should not necessarily (…)

perhaps change to ``This may happen if the TCP socket is full and thus the relay should not necessarily`` ?